packages/common/src/styles/RTLProvider.tsx

```diff
@@ -4,10 +4,6 @@ import { useRtl } from '@common/intl';
 export const RTLProvider: FC = props => {
   const isRtl = useRtl();
 
-  React.useLayoutEffect(() => {
```

Comment:
I now think it is the direction being set to `rtl` in the theme.. and this has no effect

Reply:
are you able to repro? it's still just random for me 🙄